src/main/java/ch/vorburger/exec/ManagedProcess.java

```diff
@@ -519,4 +520,8 @@ public String getProcLongName() {
                 + (executor.getWorkingDirectory() == null ? "" : " (in working directory "
                         + executor.getWorkingDirectory().getAbsolutePath() + ")");
     }
+
+    public boolean wasDestroyed() {
```

Comment:
Could you add JavaDoc here which explains exactly what this signifies? Also move it up just after isAlive().